server/src/middlewares/auth.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+import jwt from 'jsonwebtoken';
+import user from '../models';
```

Comment:
Please rename to `User` first letter uppercase. Moongose models are classes and by convention we use Pascal case convention for classes in JS